sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -125,6 +125,9 @@ async def read(
         :keyword str session_token: Token for use with Session consistency.
         :keyword dict[str, str] initial_headers: Initial headers to be sent as part of the request.
         :keyword response_hook: A callable invoked with the response metadata.
+        :keyword str priority_level: Priority based execution allows users to set a priority for each request.
```

Comment:
You have placed this docstring inbetween `:keyword response_hook:` and `:paramtype response_hook` which shouldn't be separated :)